nilearn/plotting/displays.py

```diff
@@ -260,14 +261,25 @@ def _add_lines(self, line_coords, line_values, cmap, **kwargs):
                 values of the lines.
             cmap: colormap
                 colormap used to map line_values to a color.
+            vmin: float, optional, default: None
+            vmax: float, optional, default: None
+                If not None, either or both of these values will be used to
+                as the minimum and maximum values to color lines. If None are
+                supplied the maximum absolute value within the given threshold
+                will be used as minimum (multiplied by -1) and maximum
+                coloring levels.
             kwargs: dict
                 additional arguments to pass to matplotlib Line2D.
         """
         abs_line_values_max = np.abs(line_values).max()
-        norm = colors.Normalize(vmin=-abs_line_values_max,
-                                vmax=abs_line_values_max)
+        if vmin is None:
+            vmin = -abs_line_values_max
```

Comment:
I can do whatever you think is the best case however the lack of symmetry between cases 2 and 3 that you, @lesteve . I would either go with 
* vmin is None, vmax set: vmin = -vmax (throwing a ValueError if vmin is positive) 
* vmax is None, vmin set: vmax = -vmin (throwing a ValueError if vmax is negative)

or 

* vmin set, vmax is None: vmax = np.max(np.abs(edge_data))
* vmax set, vmin is None: vmin = -np.max(np.abs(edge_data))


But you guys have the last word